### ISSUE TITLE
systemd-udev-trigger.service fails

### ISSUE THREAD

**Issue by jseun** [jseun/ddcos#1]
```
# systemctl status systemd-udev-trigger.service
● systemd-udev-trigger.service - udev Coldplug all Devices
   Loaded: loaded (/lib/systemd/system/systemd-udev-trigger.service; static)
   Active: failed (Result: exit-code) since Mon 2015-03-23 16:06:19 UTC; 1h 45min ago
     Docs: man:udev(7)
           man:systemd-udevd.service(8)
  Process: 661 ExecStart=/bin/udevadm trigger --type=subsystems --action=add (code=exited, status=203/EXEC)
 Main PID: 661 (code=exited, status=203/EXEC)

Mar 23 16:06:19 bb2f5c5729cf systemd[1]: systemd-udev-trigger.service: main process ex...XEC
Mar 23 16:06:19 bb2f5c5729cf systemd[1]: Failed to start udev Coldplug all Devices.
Mar 23 16:06:19 bb2f5c5729cf systemd[1]: Unit systemd-udev-trigger.service entered fai...te.
Warning: Journal has been rotated since unit was started. Log output is incomplete or unavailable.
Hint: Some lines were ellipsized, use -l to show in full.
```

**Comment by jseun** [comment]
```
# /bin/udevadm trigger --type=subsystems --action=add
bash: /bin/udevadm: No such file or directory
```

**Comment by jseun** [comment]
Fixed with 57229023533129cb9b3e17bc0bb1582eb37ad9dd
